config/locales/interface/input_elements/en_supply_ccus.yml

```
@@ -0,0 +1,201 @@
+---
+en:
+  input_elements:
+    share_of_industry_steel_captured_co2:
+      title: Steel industry
+      description: "What share of capturable CO<sub>2</sub> emissions in the
+        steel industry will be captured in your scenario? The amount of CO<sub>2</sub>
+        that can be captured depends on the size of the steel industry and the chosen
+        steel production technology in your scenario. You can change this
+        <a href='/scenario/demand/industry/steel'>here</a>.
```

This is nit-picky, but could we avoid having links with only "here" as the content, and instead describe what the link is pointing to?

For example:

```suggestion
        in <a href=/scenario/demand/industry/steel'>the Demand → Industry → Steel section</a>.
```

Ditto for the Dutch texts.